packages/database/src/modelClasses/Indicator.js

```diff
@@ -19,21 +19,22 @@ export class IndicatorModel extends DatabaseModel {
   }
 }
 
-const onChangeUpdateDataElement = async ({ type: changeType, record }, models) => {
-  const { code } = record;
-
+const onChangeUpdateDataElement = async (
+  { type: changeType, new_record: newRecord, old_record: oldRecord },
+  models,
+) => {
   switch (changeType) {
     case 'update':
       return models.dataSource.findOrCreate(
```

Comment:
Perhaps I should also add a `delete` of any dataSource representing the old code, if they are different. What do you think @kael89